source/xlld/wrap.d

```diff
@@ -35,17 +35,17 @@ XLOPER12 toXlOper(T, A)(in T val, ref A allocator) if(is(T == double)) {
     return ret;
 }
 
+__gshared static const toXlOperMemoryException = new Exception("Failed to allocate memory for string oper");
+__gshared static const toXlOperShapeException = new Exception("# of columns must all be the same and aren't");
```

Comment:
Exception declarations is `__gshared` now  and moved outside from generic functions to reduce number of allocations

Reply:
Maybe `immutable` is best in case of `__gshared`. The number of allocations for these exceptions is negligible though, I'm not sure it's worth it.